{
  "created_at": "2021-12-18T20:40:38.419Z",
  "title": "Third High Severity CVE in Log4j Is Published",
  "url": "https://logging.apache.org/log4j/2.x/security.html",
  "author": "niros_valtos",
  "points": 144,
  "num_comments": 99,
  "_tags": [
    "story",
    "author_niros_valtos",
    "story_29604097",
    "front_page"
  ],
  "objectID": "29604097",
  "original_created_at": "2021-12-18T12:47:48.000Z",
  "image": "https://logging.apache.org/log4j/2.x/images/ls-logo.jpg"
}